spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
```

Comment:
- [ ] Nice job setting up your tests, you are almost there! However, every test has failed showing the following error:

![Screen Shot 2022-07-19 at 7 35 15 PM](https://user-images.githubusercontent.com/83717285/179871112-49e66c64-7478-48e6-ab2a-0d57f2809d28.png)

I would suggest fixing this error, let me know if there's any information or step missing from my side in order to run the tests properly.

Reply:
@smunozmo and **to whom it may concern** 

Please, this is our last review and we have fixed all bugs. If you notice anything that needs fixing again, kindly call our attention on slack so we don't have to repeat the week because of it. Thank you.